scripts/tmp_arkode_52.jl

```diff
@@ -0,0 +1,1070 @@
+# Julia wrapper for header: arkode.h
```

Comment:
what's this scripts section for? Should be deleted?

Reply:
fixed via aa945f2